tests/Dockerfile.daemon

```diff
@@ -18,16 +20,18 @@ RUN apt update && apt install -y openssh-server
 # Set no password
 RUN sed -ie 's/^root:[^:]*:/root::/' /etc/shadow
 RUN sed -ie 's/^UsePAM/#UsePam/' /etc/ssh/sshd_config
-RUN sh -c 'echo PermitEmptyPasswords yes >> /etc/ssh/sshd_config'
-RUN sh -c 'echo PermitRootLogin yes >> /etc/ssh/sshd_config'
-
-RUN sh -c 'echo Port 22 >> /etc/ssh/sshd_config'
-RUN sh -c 'echo Port 8822 >> /etc/ssh/sshd_config'
-
-COPY --from=build /mender-install/ /
-
-RUN sh -c 'mkdir -p /var/lib/mender && echo device_type=docker-client > /var/lib/mender/device_type'
-RUN sh -c 'echo artifact_name=original > /etc/mender/artifact_info'
+RUN echo 'PermitEmptyPasswords yes\n\
+PermitRootLogin yes\n\
+Port 22\n\
+Port 8822\n' >> /etc/ssh/sshd_config
+
+COPY --from=build /mender-install/usr/bin/mender /usr/bin/mender
+COPY --from=build /mender-install/etc/mender /etc/mender
+COPY --from=build /mender-install/usr/share/mender /usr/share/mender
+COPY --from=build /mender-install/usr/share/doc/mender-client /usr/share/doc/mender-client
+COPY --from=build /mender-install/lib/systemd/system/mender-client.service /lib/systemd/system/mender-client.service
+COPY --from=build /mender-install/usr/share/dbus-1/system.d/io.mender.AuthenticationManager.conf /usr/share/dbus-1/system.d/io.mender.AuthenticationManager.conf
```

Comment:
I guess you have a reason, but this is very cumbersome to maintain. Why doesn't the original `COPY --from=build /mender-install/ /` cut it?

Reply:
that will just copy the changed files from the build stage - so everything else (e.g. `ls`, `sh`...) is gone and I have never seen the same behaviour for another image...
from what I found on looking into the ubuntu dockerfile, I _think_ the reason is a change in the ubuntu image build process (they at least planned to change to `buildah`), but couldn't track down the build steps and thus settled for the `COPY` approach...

Reply:
Ah, I see. It's less like `cp` and more like `rsync -r --delete`. :-)

Reply:
unfortunately yes... but it should be more like `cp` (and it works that way in other repos + was with the `18.04` image) 🤷‍♂️

Reply:
@kacf Exactly. IOW, the real motivation to do this was to be able to upgrade to 20.04 image... I hope this does not bit us back in the near future when adding new directories or files to the Mender installation.